main.tf

```diff
@@ -183,14 +183,14 @@ module "servers" {
   source = "./modules/nodepool"
   name   = "${local.uname}-server"
 
-  vpc_id                      = var.vpc_id
-  subnets                     = var.subnets
-  ami                         = var.ami
-  instance_type               = var.instance_type
-  block_device_mappings       = var.block_device_mappings
-  extra_block_device_mappings = var.extra_block_device_mappings
-  vpc_security_group_ids      = concat([aws_security_group.server.id, aws_security_group.cluster.id, module.cp_lb.security_group], var.extra_security_group_ids)
-  spot                        = var.spot
+  vpc_id                 = var.vpc_id
+  subnets                = var.subnets
+  ami                    = var.ami
+  instance_type          = var.instance_type
+  block_device_mappings  = var.block_device_mappings
+  extra_storage          = var.extra_storage
```

Comment:
Unless there's a good reason to change this name, this should remain `extra_block_device_mappings` to match what upstream has.

https://github.com/rancherfederal/rke2-aws-tf/blob/c40e44a0b5ff561235d19514eee34ca7dee04575/modules/agent-nodepool/main.tf#L135

https://github.com/rancherfederal/rke2-aws-tf/blob/c40e44a0b5ff561235d19514eee34ca7dee04575/modules/nodepool/main.tf#L45

Reply:
Understood and fully agree. About half way through I was trying to get the names consistent between the two ( extra_block_device_mappings & extra storage ) and when deciding which was the "original" name - picked the wrong one - and ended up switching everything to extra_storage.  

There are a few variable names that are in upstream that could be better - extra_config_config comes to mind , and depending on perspective extra_block_device_mappings might also be a candidate to be clarified ( extra_block_device_mappings ? relative to what?   extra mappings for root vol? additional storage besides root ?  etc ).   I'm considering  opening  a pr for upstream for extra_config_config (at a minimum).